test-tools/proof-explorer/src/Root.tsx

```diff
@@ -0,0 +1,1006 @@
+/* eslint-disable no-alert */
+import React, { useEffect, useState, MouseEventHandler, ChangeEventHandler, useRef, ChangeEvent } from 'react';
+import Select from 'react-select';
+import {
+    AttributeKeyString,
+    StatementTypes,
+    AccountStatementBuild,
+    AtomicStatementV2,
+    VerifiablePresentation,
+    CredentialStatements,
+    CredentialStatement,
+    ConcordiumGRPCClient,
+    streamToList,
+    ContractAddress,
+    toBuffer,
+    deserializeTypeValue,
+    SmartContractTypeValues,
+} from '@concordium/web-sdk';
+import { Buffer } from 'buffer';
+import { BrowserWalletProvider, WalletConnectProvider, WalletProvider } from './wallet-connection';
+import { GrpcWebFetchTransport } from '@protobuf-ts/grpcweb-transport';
+
+function getVerifierURL(): string {
+    return 'https://web3id-verifier.testnet.concordium.com';
```

Comment:
In case we get a request to host the front end on mainnet one day or an external user uses this front end as a starting point to create their own front end that proves something on mainnet.

Reply:
In the interest of simplicity and time, I have added this to the constants file.

Having it configurable can be done later if we really need to host this on mainnet.